### PR TITLE
whitespacecheck: Resolve most shellcheck issues

### PR DESCRIPTION
**Pull request by bergzand**
### Contribution description

This quotes a lot of variables to remove most of the issues signalled by Shellcheck. The `${IGNORE}` is left unquoted, the current script relies on that.

### Testing procedure

I've added a commit that adds some whitespace issues.

### Issues/PRs references

None

### CODE REVIEW

File: core/sched.c [hunk 1]
```diff
@@ -98,7 +98,7 @@ static inline void _set_runqueue_bit(thread_t *process)
     runqueue_bitcache |= 1 << process->priority;
 #endif
 }
-
+
```
Comment (aabadie) [inline core/sched.c:101]: unrelated :)

File: dist/tools/whitespacecheck/check.sh [hunk 2]
```diff
@@ -6,13 +6,15 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-IGNORE=`awk '{ printf ":!%s ", $0 }' $(dirname "$0")/ignore_list.txt`
+IGNORE=$(awk '{ printf ":!%s ", $0 }' "$(dirname "$0")/ignore_list.txt")
+
+echo "${IGNORE}"
```
Comment (aabadie) [inline dist/tools/whitespacecheck/check.sh:11]: Should be removed, I guess.

```suggestion
```

File: core/sched.c (hunk 1 above)
Comment (aabadie) [reply core/sched.c:101]: Ah, it's part of b01282a6de99944ee46d2bb5f3d2aa5976f0c4fa, so ok.

**Comment by bergzand** [comment]
@aabadie Is it okay to remove the test commit (b01282a) here?

**Comment by fjmolinas** [comment]
ping @aabadie!

File: dist/tools/whitespacecheck/check.sh (hunk 2 above)
Comment (bergzand) [reply dist/tools/whitespacecheck/check.sh]: Yup, removed

**Comment by aabadie** [comment]
> Is it okay to remove the test commit (b01282a) here?

Yes please!

**Comment by bergzand** [comment]
> Yes please!

Removed!

**Comment by aabadie** [comment]
Please squash!

**Comment by bergzand** [comment]
Should I also squash here?

**Review by aabadie — APPROVED** [review]
ACK
